src/nat/cli/commands/workflow/workflow_commands.py

```diff
@@ -166,12 +197,15 @@ def create_command(workflow_name: str, install: bool, workflow_dir: str, descrip
             click.echo(f"Workflow '{workflow_name}' already exists.")
             return
 
+        config_dir = new_workflow_dir / 'src' / package_name / 'configs'
+        data_dir = new_workflow_dir / 'src' / package_name / 'data'
+
         # Create directory structure
         (new_workflow_dir / 'src' / package_name).mkdir(parents=True)
```

Comment:
We can reduce some duplication:
```suggestion
        package_dir = new_workflow_dir / 'src' / package_name
        config_dir = package_dir / 'configs'
        data_dir = package_dir / 'data'
        
        # create package directory
        package_dir.mkdir(parents=True)
```

Reply:
Hi Will, thank you for enforcing these! I just made another commits to make sure we are not recomputing the path.